chord_metadata_service/experiments/models.py

```diff
@@ -0,0 +1,62 @@
+from django.db import models
+from django.core.validators import RegexValidator
+from django.contrib.postgres.fields import JSONField, ArrayField
+from chord_metadata_service.restapi.models import IndexableMixin
+
+
+class Experiment(models.Model, IndexableMixin):
+    """ Class to store Experiment information """
+
+    LIBRARY_STRATEGY = (
+        ('DNase-Hypersensitivity', 'DNase-Hypersensitivity'),
+        ('ATAC-seq', 'ATAC-seq'),
+        ('NOME-Seq', 'NOME-Seq'),
+        ('Bisulfite-Seq', 'Bisulfite-Seq'),
+        ('MeDIP-Seq', 'MeDIP-Seq'),
+        ('MRE-Seq', 'MRE-Seq'),
+        ('ChIP-Seq', 'ChIP-Seq'),
+        ('RNA-Seq', 'RNA-Seq'),
+        ('miRNA-Seq', 'miRNA-Seq'),
+        ('WGS', 'WGS'),
+    )
+
+    MOLECULE = (
+        ('total RNA', 'total RNA'),
+        ('polyA RNA', 'polyA RNA'),
+        ('cytoplasmic RNA', 'cytoplasmic RNA'),
+        ('nuclear RNA', 'nuclear RNA'),
+        ('small RNA', 'small RNA'),
+        ('genomic DNA', 'genomic DNA'),
+        ('protein', 'protein'),
+        ('other', 'other'),
+    )
+
+    # FIXME(validate qc_flags.max_length)
+    # FIXME(validate experiment_ontology_curie.max_length)
+    # FIXME(validate molecule_ontology_curie.max_length)
+    # FIXME(validation for minSize/maxSize for ArrayField)
+
+    id = models.CharField(primary_key=True, max_length=200, help_text='An arbitrary identifier for the individual.')
+
+    reference_registry_id = models.CharField(max_length=30, null=True, blank=True,
+            help_text='The IHEC EpiRR ID for this dataset, only for IHEC Reference Epigenome datasets. Otherwise leave empty.')
+    qc_flags = ArrayField(models.CharField(max_length=30, null=True, blank=True,
+            help_text='Any quanlity control observations can be noted here. This field can be omitted if empty'))
+    experiment_type = models.CharField(max_length=30, null=True, blank=True,
+            help_text="(Controlled Vocabulary) The assay target (e.g. ‘DNA Methylation’, ‘mRNA-Seq’, ‘smRNA-Seq’, 'Histone H3K4me1').")
+    experiment_ontology_curie = ArrayField(models.CharField(max_length=30, null=True, blank=True,
```

Comment:
I think we should store ontology terms in the same format we store them in other apps as:
```
{
    "id": "some_id",
    "label": "some_label"
 }
```
Then the items in ArrayField should be of JSONField type, and no regex needed.
We won't store just curies, so the field also can be renamed to `experiment_ontology`

Reply:
Can I make it simply a JSONField that would contain an array of items? Validation will be more performant that way imho.

Reply:
You can try, but I think Postgres will validate the JSONField, so you will need some property to store a list in.

Reply:
what do you mean by needing a property? a straight array is valid JSON too, so it should be OK

Reply:
oh yeah it's true, it should be ok